ajax/libs/sinon.js/package.json

```diff
@@ -16,6 +16,15 @@
     "type": "git",
     "url": "https://github.com/sinonjs/sinon"
   },
+  "npmName": "sinon",
+  "npmFileMap": [
+    {
+      "basePath": "pkg",
+      "files": [
+        "**/+(sinon.|sinon-no-sourcemaps.)*"
```

Comment:
Ah, sorry, what I wrote doesn't make sense. I thought the original code included a glob pattern, but I was wrong. Anyway, why not just have a glob of "*"? Unless there are files you want to exclude, of course.